### PR TITLE
Component/text

### PR DESCRIPTION
**Pull request by nicolobruno**
## Summary

Component Text

```
import { Text } from '@wolox/react-base-components';

<Text.h1 className={styles.title}>Some Text</Text.h1>
<Text.span className={styles.subTitle}>Some Text</Text.span>

```

### CODE REVIEW

File: lib/components/Text/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TEXT_TAGS } from './constants';
+
+function Text({ tag, elementType, children, className, formatText, dataFor, dataTip, ...props }) {
+  const Tag = tag;
```
Comment (damfinkel) [inline lib/components/Text/index.js]: you can do:

`function Text({ tag: Tag, ... })`

And you avoid this line

**Comment by FrankIglesias** [comment]
I was having some different perspective of this, something like: 
```js
import React from "react";
import { render } from "react-dom";

const h1 = props => {
  return <h1>{props.children}</h1>;
};

const h2 = props => {
  return <h2>{props.children}</h2>;
};

const Text = {
  h1,
  h2
};

const App = () => <Text.h2>Esto es un texto</Text.h2>;

render(<App />, document.getElementById("root"));

```

File: lib/components/Text/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TEXT_TAGS } from './constants';
+
+function Text({ tag: Tag, elementType, children, className, formatText, dataFor, dataTip, ...props }) {
```
Comment (FrankIglesias) [inline lib/components/Text/index.js]: What is the difference between elementType y classname, they have the same usage?

File: lib/components/Text/constants.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const TEXT_TAGS = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'span', 'label', 'p'];
```
Comment (FrankIglesias) [inline lib/components/Text/constants.js]: I'd rename this file as tags

File: lib/components/Text/index.js (hunk 2 above)
Comment (nicolobruno) [reply lib/components/Text/index.js]: Yes, in this case have same usage. I delete elementType.

**Pull request closed without merging**